### PR TITLE
chore(eslint): use common config

### PR DESCRIPTION
**Pull request by atabel**
Telefonica/js-toolbox#21

### CODE REVIEW

File: src/__screenshot_tests__/form-select-screenshot-test.tsx [hunk 1]
```diff
@@ -2,30 +2,22 @@ import {openStoryPage, screen} from '../test-utils';
...
-];
+const devices: Array<Device> = ['DESKTOP', 'MOBILE_IOS'];
 
-test.each(devices)(
-    'Select elements on the initial state appear properly on %s',
-    async (device, userAgent) => {
-        const page = await openStoryPage({
-            id: 'components-forms-select--default',
-            device,
-            userAgent,
```
Comment (atabel) [inline src/__screenshot_tests__/form-select-screenshot-test.tsx:16]: unrelated. This was a TS error. I think CI doesn't check TS on test files

File: src/__tests__/popover-test.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import Popover from '../popover';
 import {ThemeContextProvider} from '..';
 import {makeTheme} from './test-utils';
 
-type Props = Omit<React.ComponentProps<typeof Popover>, 'children' | 'description' | 'target'>;
```
Comment (atabel) [inline src/__tests__/popover-test.tsx:7]: unrelated, ts error

File: src/list.tsx [hunk 3]
```diff
@@ -246,6 +246,7 @@ interface SwitchRowContentProps extends CommonProps {
     checkbox?: undefined;
     radioValue?: undefined;
     newTab?: undefined;
+    fullPageOnWebView?: undefined;
```
Comment (atabel) [inline src/list.tsx:249]: this fixes a TS error in `list-type-test.tsx`

File: src/types/font-face.d.ts [hunk 4]
```diff
@@ -12,8 +12,12 @@ declare global {
     type FontFaceLoadStatus = 'unloaded' | 'loading' | 'loaded' | 'error';
     type FontFaceSetStatus = 'loading' | 'loaded';
 
-    interface FontFace extends FontFaceDescriptors {
-        new (family: string, source: string | ArrayBuffer, descriptors?: FontFaceDescriptors): FontFace;
+    class FontFace extends FontFaceDescriptors {
+        constructor(
```
Comment (atabel) [inline src/types/font-face.d.ts:16]: ts error

File: package.json [hunk 5]
```diff
@@ -76,6 +76,7 @@
         "@swc/core": "^1.2.93",
         "@swc/jest": "^0.2.4",
         "@telefonica/acceptance-testing": "^2.5.0",
+        "@telefonica/eslint-config": "^1.0.0-beta5",
```
Comment (atabel) [inline package.json:79]: Pending on non-beta release: https://github.com/Telefonica/js-toolbox/pull/21

**Comment by yceballost** [comment]
welcome story is broken🤭

https://mistica-web-fqpjoiea1-tuentisre.vercel.app/?path=/story/welcome-welcome--mistica

File: src/__screenshot_tests__/form-select-screenshot-test.tsx (hunk 1 above)
Comment (pladaria) [reply src/__screenshot_tests__/form-select-screenshot-test.tsx:16]: hmm, I added a `yarn ts-check` in the `ci` target... it should check types in jenkins (at least in master)

File: jest.acceptance.config.js [hunk 6]
```diff
@@ -1,4 +1,3 @@
-/* eslint-disable filenames/match-regex */
```
Comment (pladaria) [inline jest.acceptance.config.js:1]: perhaps the rule could allow any `*.config.js` files

**Review by pladaria — APPROVED** [review]

File: jest.acceptance.config.js (hunk 6 above)
Comment (atabel) [reply jest.acceptance.config.js:1]: it does

File: .swcrc [hunk 7]
```diff
@@ -6,6 +6,11 @@
         "parser": {
             "syntax": "typescript",
             "tsx": true
+        },
+        "transform": {
+            "react": {
+                "runtime": "automatic"
+            }
         }
```
Comment (atabel) [inline .swcrc:14]: this fixes the welcome story

**Comment by atabel** [comment]
> welcome story is brokenhand_over_mouth
> 
> [mistica-web-fqpjoiea1-tuentisre.vercel.app/?path=/story/welcome-welcome--mistica](https://mistica-web-fqpjoiea1-tuentisre.vercel.app/?path=/story/welcome-welcome--mistica)

Fixed

**Review by pladaria — APPROVED** [review]

**Pull request merged**